capella2polarion/data_model/work_item_attachments.py

```diff
@@ -116,9 +117,14 @@ def content_checksum(self) -> str:
             try:
                 elk_input = self.diagram.elk_input_data(self.render_params)
                 if isinstance(elk_input, tuple):
-                    input_str = ";".join(eit.json() for eit in elk_input)
+                    input_str = ";".join(
+                        eit.model_dump_json(exclude_defaults=True)
+                        for eit in elk_input
+                    )
```

Comment:
is this related to this PR? 
Does the checksum change due to the changes in context_diagrams?